scripts/rewrite/ralph-status.sh

```
@@ -0,0 +1,173 @@
+    wt_changes=$(git -C "$wt" status --short 2>/dev/null | wc -l | tr -d ' ')
+    echo "             changes  ${wt_changes} file(s) dirty"
+  fi
+
+  # 5. Open PR (if any) ---------------------------------------------------
+  local pr_json
+  pr_json=$(gh pr list --repo "$GH_REPO" --base "$BASE_BRANCH" --state open \
+              --label "$LABEL" --limit 5 --json number,title,mergeable,reviewDecision,statusCheckRollup 2>/dev/null)
+  if [ -n "$pr_json" ] && [ "$pr_json" != "[]" ]; then
+    echo ""
+    echo "  open rewrite-agent-sdk PRs on ${BASE_BRANCH}:"
+    echo "$pr_json" | python3 - <<'PY'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)